disperser/common/v2/blobstore/dynamo_metadata_store.go

```diff
@@ -354,7 +354,7 @@ func (s *BlobMetadataStore) queryBucketBlobMetadata(
 }
 
 // GetBlobMetadataByRequestedAtForward returns blobs (as BlobMetadata) in cursor range
-// (after, until] (after exclusive, until inclusive). Blobs are ordered by <RequestedAt, BlobKey>
+// (after, until) (both exclusive). Blobs are ordered by <RequestedAt, BlobKey>
```

Comment:
Making both ends exclusive, because:
- it's easier for the feed API implementation
- this is at nanosecond level (the highest resolution), so inclusive use cases can just make a `[L-1, R+1]` adjustment which will work correctly (at nanosecond, there is no time points between `L-1` and `L`)